inc/acf-gutenberg.php

```diff
@@ -0,0 +1,289 @@
+			'supports'        => $supports,
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-fifty-fifty',
+			'title'           => __( 'Fifty/Fifty Block', '_s' ),
+			'description'     => __( 'A split-content block with text and/or media.', '_s' ),
+			'render_callback' => '_s_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'editor-table',
+			'keywords'        => array( 'fifty fifty', 'columns', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => '_s_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-hero',
+			'title'           => __( 'Hero Block', '_s' ),
+			'description'     => __( 'A hero with an optional call to action.', '_s' ),
+			'render_callback' => '_s_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'slides',
+			'keywords'        => array( 'hero', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => '_s_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-recent-posts',
+			'title'           => __( 'Recent Posts Block', '_s' ),
+			'description'     => __( 'A set of recent posts displayed by category and/or tag.', '_s' ),
+			'render_callback' => '_s_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'admin-page',
+			'keywords'        => array( 'recent posts', 'posts', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => '_s_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+		)
+	);
+
+	acf_register_block_type(
+		array(
+			'name'            => 'wds-related-posts',
+			'title'           => __( 'Related Posts Block', '_s' ),
+			'description'     => __( 'A set of manually selected posts.', '_s' ),
+			'render_callback' => '_s_acf_block_registration_callback',
+			'category'        => 'wds-blocks',
+			'icon'            => 'admin-page',
+			'keywords'        => array( 'related posts', 'posts', 'wds' ),
+			'mode'            => 'preview',
+			'enqueue_assets'  => '_s_acf_enqueue_backend_block_styles',
+			'align'           => 'wide',
+			'supports'        => $supports,
+		)
+	);
+}
+add_action( 'acf/init', '_s_acf_init' );
+
+/**
+ * Our callback function – this looks for the block based on its given name.
+ * Name accordingly to the file name!
+ *
+ * @param array $block The block details.
+ * @return void Bail if the block has expired.
+ * @author Corey Collins
+ */
+function _s_acf_block_registration_callback( $block ) {
+
+	// Convert the block name into a handy slug.
+	$block_slug = str_replace( 'acf/', '', $block['name'] );
+
+	// If the block has expired,then bail!
+	if ( _s_has_block_expired(
```

Comment:
When a block is scheduled in the future (or has expired), it no longer shows up in the block editor, until you make a change. I think the block should appear in the block editor, no matter what.

![screenshot](https://dl.dropbox.com/s/9xe36zatu9ngi34/Screenshot%202019-06-14%2015.14.54.gif?dl=0)

Reply:
Good call. Even better hero.